pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -604,3 +604,8 @@ func DeleteIPsecEncryptRoute() {
 		}
 	}
 }
+
+func DeleteIPSecXfrmSpi(spi uint8) {
+	time.Sleep(linux_defaults.IPsecKeyDeleteDelay)
+	ipsecDeleteXfrmSpi(spi)
+}
```

Comment:
To delete the rules, I'm using ipsecDeleteXfrmSpi() but that function isn't exported, so I've created a wrapper function to call it.